guibot/finder.py

```diff
@@ -2019,7 +2019,7 @@ def __synchronize_backend(self, backend=None, category="text", reset=False):
             if backend == "pytesseract":
                 import pytesseract
                 self.ocr = pytesseract
-                self.ocr_config = r"--tessdata-dir %s --oem %s --psm %s "
+                self.ocr_config = r"--tessdata-dir '%s' --oem %s --psm %s "
```

Comment:
Awesome, and as this is a test boundary it is ok to go like this.

Reply:
Hey @pevogam, 

I hope I did the rebasing as suggested - I'm quite a beginner to pull requests. The commit name should be changed.